paddle/fluid/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 add_subdirectory(memory)
 add_subdirectory(platform)
 add_subdirectory(framework)
+if(WITH_PYTHON)
+  add_subdirectory(imperative)
```

Comment:
It is very strange to use an adjective word as a module name. Module name should be noun.

Reply:
Well, ```imperative``` is a temporal name, because @panyx0718 and I didn't come up with a good one..